setup.py

```diff
@@ -81,7 +81,7 @@ def run(self):
         "netcdf4",
         "pyyaml",
         "pyshp",
-        "scipy<1.3",
```

Comment:
Could you tell where to do it? I met this problem when i use Jupyternote book?

Reply:
@TR160qg *landlab* does not require a particular version of *scipy*. What you are looking at here is an old pull request that removes the version requirement.

If you've encountered a problem using *landlab*, please submit an [issue](https://github.com/landlab/landlab/issues) describing your problem. My guess, if *landlab* is requiring an older version of *scipy*, is that you are using an older version of *landlab* and you could try updating it.